assets/airports/egcc.json

```diff
@@ -0,0 +1,582 @@
+		"D181L":	["N53d32m64.000", "W002d05m85.000"],
+		"D188X":	["N53d20m78.000", "W002d10m02.000"],
+		"D220I":	["N53d37m54.000", "W002d15m53.000"],
+		"D220P":	["N53d32m03.000", "W002d22m76.000"],
+		"D255H":	["N53d19m05.000", "W002d28m49.000"],
+		"D255L":	["N53d17m85.000", "W002d34m85.000"],
+		"D255M":	["N53d17m55.000", "W002d36m45.000"],
+		"D255P":	["N53d16m65.000", "W002d41m21.000"],
+		"D299G":	["N53d24m45.000", "W002d26m38.000"],
+		"DALEY":	["N53d40m16.000", "W002d20m57.000"],
+		"DAYNE":	["N53d14m19.000", "W002d01m45.000"],
+		"DESIG":	["N53d31m38.000", "W001d53m34.000"],
+		"EKLAD":	["N53d15m14.000", "W002d49m29.000"],
+		"GASKO":	["N54d13m29.000", "W001d57m21.000"],
+		"GIGTO":	["N53d38m00.000", "W004d05m09.000"],
+		"GOLES":	["N53d36m29.000", "W001d05m00.000"],
+		"HON58":	["N53d12m79.000", "W002d20m05.000"],
+		"IBRAR":	["N53d31m09.000", "W003d41m02.000"],
+		"IOM":		["N54d04m00.000", "W004d45m08.000"],
+		"KELLY":	["N53d54m03.000", "W004d21m09.000"],
+		"KUXEM":	["N53d15m11.000", "W002d40m47.000"],
+		"LAKEY":	["N54d14m03.000", "W002d58m09.000"],
+		"LESTA":	["N52d44m27.000", "W001d04m19.000"],
+		"LIBSO":	["N53d31m29.000", "E000d05m36.000"],
+		"LISTO":	["N53d08m36.000", "W002d11m57.000"],
+		"LYNAS":	["N53d26m06.000", "W004d20m00.000"],
+		"MAKUX":	["N53d58m05.000", "W004d52m05.000"],
+		"MALUD":	["N53d24m08.000", "W003d36m05.000"],
+		"MCT":		["N53d21m25.000", "W002d15m44.000"],
+		"MCT06":	["N53d26m77.000", "W002d20m28.000"],
+		"MCT07":	["N53d24m38.000", "W002d26m41.000"],
+		"MCT08":	["N53d25m69.000", "W002d27m13.000"],
+		"MCT12":	["N53d22m31.000", "W002d14m26.000"],
+		"MIRSI":	["N53d32m03.000", "W002d42m07.000"],
+		"MONTY":	["N52d53m34.000", "W003d10m26.000"],
+		"OKTEM":	["N52d43m10.000", "W003d11m15.000"],
+		"OTBED":	["N53d17m17.000", "E000d01m55.000"],
+		"PENIL":	["N53d37m00.000", "W003d39m08.000"],
+		"PO28A":	["N53d16m79.000", "W002d10m66.000"],
+		"POL":		["N53d44m38.000", "W002d06m12.000"],
+		"POL33":	["N53d11m83.000", "W002d11m44.000"],
+		"POL34":	["N53d16m81.000", "W002d38m90.000"],
+		"POL36":	["N53d15m97.000", "W002d42m65.000"],
+		"REXAM":	["N53d04m00.000", "W003d09m06.000"],
+		"ROSUN":	["N53d40m08.000", "W002d20m57.000"],
+		"SANBA":	["N53d08m22.000", "W002d20m03.000"],
+		"SETEL":	["N54d00m45.000", "W002d26m09.000"],
+		"SONEX":	["N53d29m53.000", "W002d10m21.000"],
+		"SOSIM":	["N53d48m09.000", "W004d30m05.000"],
+		"TABLY":	["N53d16m19.000", "W002d27m02.000"],
+		"TILNI":	["N54d32m51.000", "W001d51m18.000"],
+		"TNT":		["N53d03m14.000", "W001d40m12.000"],
+		"UPTON":	["N53d35m13.000", "W001d18m03.000"],
+		"WAL":		["N53d23m05.000", "W003d08m04.000"],
+		"WAL33":	["N53d29m51.000", "W002d13m77.000"],
+		"XOBRO":	["N53d28m17.000", "W002d25m18.000"],
+		"XUMAT":	["N53d27m95.000", "W002d28m38.000"],
+		"_05L":		["N53d20m51.200", "W002d17m15.950"],
+		"_05R":		["N53d19m55.100", "W002d18m38.380"],
+		"_23L":		["N53d21m40.750", "W002d15m33.410"],
+		"_23R":		["N53d20m53.350", "W002d16m37.950"],
```

Comment:
These two fixes are backwards. It looks like `"_23L"` and `"_23R"` should be switched.